Assets/Scripts/Models/Character.cs

```diff
@@ -376,7 +376,7 @@ public void WriteXml(XmlWriter writer)
         foreach (Need n in needs)
         {
             int storeAmount = (int)(n.Amount * 10);
-            needString = needString + n.needType + ";" + storeAmount.ToString() + ":";
+            needString = needString + n.Type + ";" + storeAmount.ToString() + ":";
```

Comment:
We are using ObjectType for Furniture and Inventory, StatType for Stats and Type for Needs, and maybe others for other clases. I think we need some sort of standard. Is it Type or ObjectType for all, or ClassNameType?